etl/steps/data/garden/war/2023-07-07/mars.meta.yml

```diff
@@ -0,0 +1,66 @@
+dataset:
+  title: History of war (Project Mars, 2020)
+  description: >-
+    This dataset contains information on armed conflicts, both internal and external, in the period of 1800 and 2011.
+
+
+    It has been constructed using the Project Mars dataset by Lyall, Jason.
+
+
+    More details about Project Mars:
+
+
+tables:
+  mars:
+    # (Inherited from meadow, remove if not different.)
+    variables:
+      number_deaths_ongoing_conflicts_high:
+        title: Number of soldier deaths in ongoing conflicts (high estimate)
+        unit: deaths
+        description: >-
+          The number of soldiers killed in a year over the course of wars. Excluded from this total are missing soldiers as well as those who
+          died from illness and/or disease. This is the low estimate of those who were killed.
+        display:
+          numDecimalPlaces: 0
+      number_deaths_ongoing_conflicts_low:
+        title: Number of soldier deaths in ongoing conflicts (low estimate)
+        unit: deaths
+        description: >-
+          The number of soldiers killed in a year over the course of wars. Excluded from this total are missing soldiers as well as those who
+          died from illness and/or disease. This is the high estimate of those who were killed.
+        display:
+          numDecimalPlaces: 0
+
```

Comment:
Is there no central estimates available?

Reply:
I see there aren't. But I was thinking about how to plot this in the future (maps, for example). Maybe separate, or create an average between both estimations.